src/components/voltage-level-choice.js

```diff
@@ -10,16 +10,13 @@ import makeStyles from '@mui/styles/makeStyles';
 import Menu from '@mui/material/Menu';
 import MenuItem from '@mui/material/MenuItem';
 import ListItemIcon from '@mui/material/ListItemIcon';
-
 import ListItemText from '@mui/material/ListItemText';
 import Button from '@mui/material/Button';
 import Typography from '@mui/material/Typography';
-import { useSelector } from 'react-redux';
 import { getNominalVoltageColor } from '../utils/colors';
-import { PARAM_USE_NAME } from '../utils/config-params';
-import { getNameOrId } from './diagrams/singleLineDiagram/utils';
+import { useNameOrId } from './util/equipmentInfosHandler';
 
-const useStyles = makeStyles((theme) => ({
```

Comment:
do not remove theme here, it's better to use theme.spacing(1) etc... than raw values for example, even if it's not use for the moment...
I understand it's a cleaner code if not used, maybe in another PR ?